infor-design/enterprise
# 3041 - Fix incorrect disabled/readonly text colors due to CSS changes

**Explain the _details_ for making this change. What existing problem does the pull request solve?**
This PR fixes a handful of issues found by Q/A after the changes in #3041.

**Related github/jira issue (required)**:
Fixes #3041
Fixes #3199

**Steps necessary to review your pull request (required)**:
Pull this branch, build, and run the demoapp. Then...

_To test #3041:_
Double check the placeholder color text on disabled and readonly fields in the following example pages.  Specifically, check Mac OSX Safari:
- Standard Inputs: http://localhost:4000/components/input/example-index.html?theme=uplift&variant=contrast
- Colorpicker (Soho): http://localhost:4000/components/colorpicker/test-states.html
- Colorpicker (Vibrant): http://localhost:4000/components/colorpicker/test-states.html?theme=uplift&variant=contrast
- File Uploader (Soho): http://localhost:4000/components/fileupload/example-index.html?theme=soho&variant=light
- File Uploader (Vibrant): http://localhost:4000/components/fileupload/example-index.html?theme=uplift&variant=contrast
- Listview Searchfield: http://localhost:4000/components/listview/test-search-disabled.html?theme=uplift&variant=contrast
- Personalized Searchfield: http://localhost:4000/components/personalize/example-index.html?theme=uplift&variant=contrast&colors=9279a6
- Spinbox: http://localhost:4000/components/spinbox/example-disabled.html

_To test #3199:_
- Open http://localhost:4000/examples/saleshub/edit-profile.html?theme=soho&variant=dark
- Ensure the background color on the Builder Actions area is not bright white.
